### ISSUE TITLE
Be able to template call activies

### ISSUE THREAD

**Issue by nikku** [bpmn-io/bpmn-js-properties-panel#704]
__Is your feature request related to a problem? Please describe__

C8 element templates do not offer support to bind the [called element](https://docs.camunda.io/docs/components/modeler/bpmn/call-activities/#xml-representation) via `zeebe:calledElement`. In order to ship call activities as re-usable building blocks we'd need to add support for that.

The existing input and output mapping features can be re-used to define mappings into and out of the called process.

__Describe the solution you'd like__

* [ ] A new template binding `zeebe:calledElement:processId` allows you to establish the binding ([`zeebe:calledElement#processId`](https://docs.camunda.io/docs/components/modeler/bpmn/call-activities/#xml-representation)). The binding may be FEEL, but does not have to.

__Describe alternatives you've considered__

None.

__Additional context__

Related to [this conversation](https://camunda.slack.com/archives/C0350959WG1/p1654859357742999).

**Comment by nikku** [comment]
An initial exploration looks promising but also found that there is a little bit more to do: 

Given [this template](https://github.com/bpmn-io/bpmn-js-properties-panel/files/8879422/misc.json.txt)

You can see how input can be used in a straight forward manner.

What is missing is support for dynamic output mapping (which is not an engine feature at the moment). Instead we'd need to statically map all possible execution results to assign them (optionally) to variables.

What is also missing is an icon to call apart element template backed call activities from normal ones.

----

![image](https://user-images.githubusercontent.com/58601/173080034-4c69eb5a-d1d9-4783-abb4-f1ac712d4d72.png)

**Comment by pinussilvestrus** [comment]
One confirmation that this one is needed: https://github.com/camunda/camunda-modeler/issues/3006.

**Comment by Rado-1** [comment]
Definitely, binding properties to the processId and propagateAllChildVariables attributes is a needed feature. Icon for call activity would also be helpful. When we could expect its implementation?

**Comment by nikku** [comment]
I'm not sold on `propagateAllChildVariables` being templated @Rado-1. The purpose is to be explicit (with your information requirements).

I added it to the issue description regardless.

**Comment by Rado-1** [comment]
> I'm not sold on `propagateAllChildVariables` being templated

Basically, each parameter should be templatable. `propagateAllChildVariables`  can be used, e.g.,  in a case of reusable sub-process which uses local variables to report/convey status of own processing to the "parent" process. So the decision weather to use `propagateAllChildVariables` (or any other parameter) should be rather on template developer and should not be restricted by technology.

**Comment by ingorichtsmeier** [comment]
Hi team, I've lately recommended call activities with templates for a smooth migration from Camunda 7 to Camunda 8. Right now, the customer uses an execution listener to initialize variables for the called process.

In the future, a preconfigured Call Activity would be the way to go. Could you please move it higher on the priorities?

The requirement is, that the template defines the called process as read-only or hidden and give required variables for the called process. The business analyst should enter the values as expressions to the predefined variables. 

Output mapping may be the other way around: The call activity defines the variables with the result of the subprocess and the business analyst fills the variable names for the super process.

**Comment by barmac** [comment]
I am transferring this issue to bpmn-io/bpmn-js-element-templates as it belongs to the templates repo.
